INN/umbrella-caribbean
# Pre-footer newsletter signup area

## Changes

This pull request makes the following changes:

- Adds styling for the newsletter pre-footer signup area

Desktop:
![Screen Shot 2019-12-04 at 9 03 07 AM](https://user-images.githubusercontent.com/18353636/70148838-f293a080-1674-11ea-90f2-00a54fdc11ce.png)

Mobile:
![Screen Shot 2019-12-04 at 9 03 19 AM](https://user-images.githubusercontent.com/18353636/70148837-f1fb0a00-1674-11ea-8965-6c77cbf9326b.png)

## Why



For #10

## Testing/Questions

Features that this PR affects:

- Pre-footer newsletter area


Questions that need to be answered before merging:


Steps to test this PR:

1. Turn on the pre-footer widget area in theme options under Layout -> Sidebar Options
2. Create a test GF signup form with a name field and an email field
3. Add an image widget to the "Before Footer" widget area and add an image to it
4. Add a text widget to the "Before Footer" widget area and make the title "Newsletter Signup" and paste this text: 
>Even if the most intense reverberations from Commissioner James O’Neill’s August 19 decision subside, the Garner case will continue to shape conversations about the mayor, the police department and the system in place to oversee cops. Even if the most intense reverberations from Commissioner James O’Neill’s August 19 decision subside, the Garner case will continue to shape conversations about the mayor, the police department and the system in place to oversee cops.
5. Add the GF widget to the "Before Footer" widget area and select your test form. Don't display title or description. Give the form widget an empty space in the title
6. View the frontend and make sure it looks ok
